### PR TITLE
fix quickstart

### LINKED ISSUE #191: Issues with Quick Start
Having gone through the [Quick Start](https://docs.kilt.io/docs/develop/sdk/quickstart), I've come across a few issues and wanted to use this to flag them and follow-up with a PR.

- [x] Instruct users to run `npm init -y` after making a new directory
- [x] Where it says "We will use the official Spiritnet address" `return Kilt.connect('wss://spiritnet.kilt.io/')` it should be `const api = await Kilt.connect('wss://spiritnet.kilt.io/')`
- [x] After testing a few things, the IPFS hosted web3name (`john_doe`) details is causing issues, it may be better to use another Gist hosted one instead
- [x] `await Kilt.Did.resolve(johnDoeDid)` should be `uri` instead of `johnDoeDid`
- [x] The last steps that decode the credential datastructure seems to have issues. But this could be related to the web3name being used. Here's what I had to get it working:

```js
const {
        data: credential,
      } = await axios.get(endpoints[0].serviceEndpoint[0])

      const {rootHash} = credential
      
      const encodedAttestationInfo = await api.query.attestation.attestations(
        rootHash
      )
      
      // This function will throw if the attestation does not exist.
      const attestationInfo = await Kilt.Attestation.fromChain(
        encodedAttestationInfo,
        rootHash
      )
  ```
  
  Not sure what changes to propose for the last point, the request hangs for a while. Is this normal?

### PR DESCRIPTION
**Pull request by Dudleyneedham**
## fixes https://github.com/KILTprotocol/docs/issues/191

Updating the quickstart guide to include js/ts initialisation of a project.
Additionally, adding the latest SDK changes to connections.
Changing the naming `john_doe` to `uri`
Fixed the decoding and altered the order of verification.

## How to test:
Run through the quickstart, it should take 5 minutes.

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

**Comment by Dudleyneedham** [comment]
@sacha-l I have updated according to what we discussed.

File: docs/develop/01_sdk/01_quickstart.md [hunk 1]
```diff
@@ -61,9 +63,36 @@ For this, we run `mkdir kilt-rocks && cd kilt-rocks`.
   </TabItem>
 </Tabs>
 
+<Tabs groupId="ts-js-choice">
+  <TabItem value='ts' label='Typescript' default>
+
+After you have imported the SDK you will be able to access the functionalities that KILT provides.
```
Comment (ntn-x2) [inline docs/develop/01_sdk/01_quickstart.md]: Can you indent this with a tab? And the same for all the other paragraphs?

File: docs/develop/01_sdk/01_quickstart.md [hunk 2]
```diff
@@ -61,9 +63,36 @@ For this, we run `mkdir kilt-rocks && cd kilt-rocks`.
...
+
+  ```bash
+  touch tsconfig.json
```
Comment (ntn-x2) [inline docs/develop/01_sdk/01_quickstart.md:72]: Maybe just a line before this explaining that an empty file with this name needs to be created.

File: docs/develop/01_sdk/01_quickstart.md [hunk 3]
```diff
@@ -178,27 +208,28 @@ We can select one of the endpoints and query the URL to see if it returns a KILT
...
-  {VerifyAttestation}
+  {VerifyCredential}
 </SnippetBlock>
 
-Execute the script and see if you get a valid attestation for John Doe's credential!
+Now excute the script wait to see whether we can successfully retrieve **and** verify one of John Doe's credentials!
 
-If so, it is then time to verify the credential.
-This will be indicated by the result of the **verification** process as shown in the snippet below:
+Now time to query the credential's `rootHash` from the blockchain to see if it has been **attested** by someone:
```
Comment (ntn-x2) [inline docs/develop/01_sdk/01_quickstart.md]:
```suggestion
Now it's time to query the credential's `rootHash` from the blockchain to see if it has been **attested** by someone:
```

File: code_examples/core_features/getting_started/07_verify_attestation.ts [hunk 4]
```diff
@@ -18,5 +18,9 @@ export async function main(
   )
 
   // Return false if attestation.revoked is true, or true otherwise.
-  return !attestationInfo.revoked
+  const revokedStatus = !attestationInfo.revoked
+
+  console.log('Checking the revoked status of the attestation', revokedStatus)
```
Comment (ntn-x2) [inline code_examples/core_features/getting_started/07_verify_attestation.ts]:
```suggestion
  console.log('Checking the revocation status of the attestation', revokedStatus)
```

**Review by ntn-x2 — APPROVED** [review]
Apart from few minor nitpicks, looks good to me!

File: docs/develop/01_sdk/01_quickstart.md (hunk 1 above)
Comment (Dudleyneedham) [reply docs/develop/01_sdk/01_quickstart.md]: Done

**Pull request merged**